src/domain/hooks/helpers/event-cache.helper.ts

```diff
@@ -163,6 +147,29 @@ export class EventCacheHelper implements OnModuleInit, OnModuleDestroy {
     await this.cacheService.increment(cacheKey, MAX_TTL);
   }
 
+  /**
+   * Logs the number of unsupported chain events for each chain and clears the store.
+   * This function is public just for testing purposes.
+   */
+  @Interval(EventCacheHelper.UNSUPPORTED_EVENTS_LOG_INTERVAL)
```

Comment:
What do you think about importing [the readable value](https://github.com/nestjs/schedule/blob/master/lib/enums/cron-expression.enum.ts#L6) and setting it as the constant? (I didn't know the package accepted milliseconds.)

Reply:
I've replaced the annotation to `@Cron(CronExpression.EVERY_MINUTE)` I think it's more readable and meaningful as you suggested. Thanks!

[5d51919](https://github.com/safe-global/safe-client-gateway/pull/1990/commits/5d519192550a0547ff1e9f01ed2f460744e0c6a0)